ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_activate.sh

```diff
@@ -43,7 +43,7 @@ function osd_activate {
   else
     OSD_WEIGHT=$(df -P -k "$OSD_PATH" | tail -1 | awk '{ d= $2/1073741824 ; r = sprintf("%.2f", d); print r }')
   fi
-  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION}"
+  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION[@]}"
```

Comment:
osd_common.sh and osd_directory.sh should also be updated to expand the CRUSH_LOCATION array (by adding  `[@]`).